crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -478,6 +478,20 @@ fn cache_latest_events(room: &Room, room_info: &mut RoomInfo, events: &[SyncTime
                     // older.
                     break;
                 }
+                PossibleLatestEvent::YesPoll(_) => {
```

Comment:
This branch is exactly the same as the `PossibleLatestEvent::YesMessageLike(_)` branch. To avoid code duplication, you may want to refactor as the following:

```rust
PossibleLatestEvent::YesMessageLike(_) | PossibleLatestEvent::YesPoll(_) => { … }
```

Reply:
Yes indeed, thanks!